docs/eclipse/release-notes.md

```diff
@@ -5,6 +5,29 @@ title: Release Notes
 
 Here you will find the latest information about releases of the jSparrow Eclipse plugin.
 
+## 3.22.0 <Badge vertical="middle" text="15.10.2020"/>
+
+The [86<sup>th</sup> refactoring rule](/rules/) is shipped with jSparrow 3.20.0. It makes use of the [`Files.newBufferedWriter`](https://docs.oracle.com/javase/7/docs/api/java/nio/file/Files.html#newBufferedWriter(java.nio.file.Path,%20java.nio.charset.Charset)) method for initializing BufferedWriter objects for writing to text to files.
```

Comment:
The provided link [1] for `Files.newBufferedWriter` does not seem to have the correct anchor. The correct URL should be [2]. 

[1] https://docs.oracle.com/javase/7/docs/api/java/nio/file/Files.html#newBufferedWriter(java.nio.file.Path,%20java.nio.charset.Charset)
[2] https://docs.oracle.com/javase/7/docs/api/java/nio/file/Files.html#newBufferedWriter(java.nio.file.Path,%20java.nio.charset.Charset,%20java.nio.file.OpenOption...)